File: 1863. Sum of All Subset XOR Totals.java
class Solution {
    public int subsetXORSum(int[] nums) {
      int or=0;
      for(int i = 0; i<nums.length; i++ ){
        or |= nums[i];
      }
      int result = or * (int)Math.pow(2, nums.length-1);
      return result;
    }
}

File: 1863_Sum_of_All_Subset_XOR_Totals.rs
impl Solution {
    pub fn subset_xor_sum(nums: Vec<i32>) -> i32 {
        let mut or = 0;
        for &i in nums.iter(){
           or |=  i;
        }
        or * 2_i32.pow((nums.len() - 1) as u32)
    }
}
